### ISSUE TITLE
Add sign up to Monitor special action 

### ISSUE THREAD

**Issue by glogiotatidis** [mozmeao/snippets-service#1110]
Related ASR PR https://github.com/mozilla/activity-stream/pull/5220/files/5542e7424581c54cf5ca3a60477fa385fab81021

See also #1102 

## Check list
 - [x] Confirm with FxA team that flow metrics used are OK

**Comment by jgmize** [comment]
After deploying #1103 to production, @jkt230 and I created a test snippet with a button and the [URL from the example docs](TODO). Clicking on the button took me to the URL, but when I tried to sign in I got the following error:

![image](https://user-images.githubusercontent.com/161718/63372125-1db6e600-c34b-11e9-8202-b2c546447e15.png)

**Comment by jgmize** [comment]
To resolve the issue above we needed to change the dynamic `utm_content=[[channels]]` part of the url to a temporarily hard coded `utm_content=nightly`. The targeting section of the admin seems to have gone missing as part of the deployed changes.

**Comment by jgmize** [comment]
Jean & I figured out that the targeting section of the admin got moved to the Jobs section, so this is a non-issue for snippets that get published by Jobs. I granted Job permissions to the `ASR Snippets` group so that Jean and other ASR Snippets members can view & add jobs.

**Comment by jkt230** [comment]
Actually tried signing in with an email address on a snippet that has a Job published with nightly targeting which left the utm_content=[[channels]] and still received this error:
![Screen Shot 2019-08-23 at 2 41 28 PM](https://user-images.githubusercontent.com/6807556/63625355-383fc800-c5b4-11e9-9d36-cb5cd226771f.png)

**Comment by glogiotatidis** [comment]
I cannot say how wrong is that you pushed to production without first consulting me or reading the issues of the code pushed to production. #1034 had a number of things to check before reaching prod and a number of migration steps that haven't happened.  I'm sure that this happened with the best intentions but still not ok.

Here you're trying to test an open issue without any related code merged. I'm not surprised that it doesn't work. 

ASR didn't have this merged in firefox before I got on PTO and thus I set a reminder to fix this by the time 71 hits Beta.
